src/test/ui/array_const_index-0.stderr

```diff
@@ -8,5 +8,20 @@ LL | const B: i32 = (&A)[1];
    |
    = note: #[deny(const_err)] on by default
 
-error: aborting due to previous error
+error: any use of this value will cause an error
+  --> $DIR/array_const_index-0.rs:12:1
+   |
+LL | const B: i32 = (&A)[1];
+   | ^^^^^^^^^^^^^^^-------^
+   |                |
+   |                index out of bounds: the len is 0 but the index is 1
```

Comment:
Why do we error twice now for the same thing, where we only errored once before?

Reply:
I presume because the query is once called with `Reveal::UserFacing` and once with `Reveal::All`